client/tools/rhncfg/config_management/rhncfg_get.py

```diff
@@ -92,28 +92,30 @@ def run(self):
                 dep_trans.add_preprocessed(f, temp_file, info, dirs_created, strict_ownership=0)
                 continue
             elif info.get('filetype') == 'symlink':
-                print "%s -> %s" % (info['path'], info['symlink'])
+                print("%s -> %s" % (info['path'], info['symlink']))
                 continue
             elif info.get('filetype') == 'directory':
-                print "%s is a directory entry, nothing to get" % info['path']
+                print("%s is a directory entry, nothing to get" % info['path'])
                 continue
             else:
-                print open(temp_file).read()
+                print(open(temp_file).read())
                 os.unlink(temp_file)
 
         if topdir:
             try:
                 dep_trans.deploy()
-            except Exception, e:
+            except Exception:
+                e = sys.exc_info()[1]
                 try:
                     dep_trans.rollback()
-                except FailedRollback, e2:
+                except FailedRollback:
+                    e2 = sys.exc_info()[1]
```

Comment:
Look at line below: it doesn't work for Python 3. use our own function raise_with_tb